src/services/contracts/acbuMinting.service.ts

```
@@ -298,6 +360,17 @@ export class MintingService {
       const feeRate = ContractClient.fromScVal(result);
       return Number(feeRate);
     } catch (error) {
+      if ((params as any).txId) {
+        try {
+          await prisma.transaction.update({
+            where: { id: (params as any).txId },
+            data: { status: "FAILED" }
```

ESLint

[error] Insert `,`

(prettier/prettier)